### PR TITLE
openexr: added 3.2.1

### PR DESCRIPTION
**Pull request by kenmays**
OpenEXR 3.2.1 - build tested on Haiku R1B4.
- Removed OpenEXR 2.5.4 (not in use by any known packages)
- Ref: https://dev.haiku-os.org/ticket/18537

Migrate to OpenEXR 3.2.1 (Haiku R1B5, Haiku >=hrev57475):
- haiku_datatranslators

Builds verified: w/OpenEXR 3.2.1
- [x] libjxl 
- [x] alembic
- [x] openimageio
- [x] opencolorio
- [x] blender3
- [x] haikuwebkit

### CODE REVIEW

File: media-libs/openexr/openexr32-3.2.1.recipe
Comment (jmairboeck) [inline media-libs/openexr/openexr32-3.2.1.recipe:1]: rename this to `openexr3.2` maybe?

Comment (kenmays) [reply media-libs/openexr/openexr32-3.2.1.recipe:1]: Resolved

**Comment by Begasus** [comment]
@waddlesplash good to go?

**Comment by korli** [comment]
This shouldn't be merged, haiku depends on libilmimf_2_4, see https://cgit.haiku-os.org/haiku/tree/src/data/package_infos/generic/haiku_datatranslators#n36

**Comment by Begasus** [comment]
@korli keeping 2.4.1 as openexr2.4-2.4.1.recipe would be good then? Making the 3.* version the default in here shouldn't effect Haiku (and base packages don't conflict).

**Comment by korli** [comment]
> @korli keeping 2.4.1 as openexr2.4-2.4.1.recipe would be good then? Making the 3.* version the default in here shouldn't effect Haiku (and base packages don't conflict).

Yes, supposing everything that depends on openexr 3.0 can be rebuilt with openexr 3.2

**Comment by korli** [comment]
@kenmays any chance to get this right for a merge?

**Comment by korli** [comment]
No chance to merge this however.

**Comment by kenmays** [comment]
@Begasus - I'll close this for your newer PR:[ #10165](https://github.com/haikuports/haikuports/pull/10165)

**Comment by Begasus** [comment]
My bad here @kenmays sorry 'bout that, needed this one to checkout something else. :)

**Pull request closed without merging**